content/index.html.erb

```diff
@@ -91,42 +91,47 @@ <h2 class="pa0 pb1">Popular articles</h2>
   </div>
 </div>
 
+<% featured = blog_articles %>
```

Comment:
Why do you alias `blog_articles` with `featured` instead of using `blog_articles` directly? Maybe `blog_articles` is not a good name? ;)

Reply:
I didnt want it to make the network request to the blog more than once. Not sure if that really matters since this is at compile time and not run time, but yeah...